lab-carolina/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "hpptserver",
+  "version": "1.0.0",
+  "description": "lab07 http server",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "carolina ceja",
+  "license": "ISC",
+  "devDependencies": {
+    "cowsay": "^1.1.8",
```

Comment:
Cowsay should not be in the devDeps, given that it's a requirement for your http server to function correctly. 

You're also missing your mocha/chai deps here.